src/strategies/stablexswap/index.ts

```diff
@@ -0,0 +1,79 @@
+import { formatUnits } from '@ethersproject/units';
+import { multicall } from '../../utils';
+import { strategy as erc20BalanceOfStrategy } from '../erc20-balance-of';
+
+export const author = 'stablexswap';
+export const version = '0.0.1';
+
+const abi = [
+  {
+    inputs: [
+      {
+        internalType: 'address',
+        name: '',
+        type: 'address'
+      }
+    ],
+    name: 'userInfo',
+    outputs: [
+      {
+        internalType: 'uint256',
+        name: 'amount',
+        type: 'uint256'
+      }
+    ],
+    stateMutability: 'view',
+    type: 'function'
+  }
+];
+
+export async function strategy(
+  space,
+  network,
+  provider,
+  addresses,
+  options,
+  snapshot
+) {
+  const blockTag = typeof snapshot === 'number' ? snapshot : 'latest';
+  const score = await erc20BalanceOfStrategy(
+    space,
+    network,
+    provider,
+    addresses,
+    options,
+    snapshot
+  );
+
+  const balances = await Promise.all(
+    options.staxAddresses.map((staxAddress) =>
+      multicall(
+        network,
+        provider,
+        abi,
+        addresses.map((address: any) => [
+          staxAddress,
+          'userInfo',
+          [address],
+          { blockTag }
```

Comment:
This line will not be used, you can remove it.

Reply:
is this referring to blocktag?

Reply:
have not made this edit yet because I couldn't tell which line(s) should be removed, sorry for the confusion

Reply:
Yes i mean the line with blockTag
```suggestion
```

Reply:
done

Reply:
I see that its redundant now, make sense